backend/app/tests/crud/collections/test_crud_collection_delete.py

```
@@ -51,14 +53,21 @@ def test_cannot_delete_others_collections(self, db: Session):
 
     @openai_responses.mock()
     def test_delete_document_deletes_collections(self, db: Session):
-        store = DocumentStore(db)
+        project = get_project(db)
+        store = DocumentStore(db, project_id=project.id)
         documents = store.fill(1)
 
+        stmt = select(APIKey).where(
+            APIKey.project_id == project.id, APIKey.is_deleted == False
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not APIKey.is_deleted:` for false checks

Replace with `not APIKey.is_deleted`

(E712)